### PR TITLE
[PWD-2603] CT Dashboard Fixes - Fixed Program Detail Page

### PR DESCRIPTION
**Pull request by hiljaker**
### What type of PR is this? (_check all applicable_)

- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

**Review by FahemHakikiKhaya — APPROVED** [review]

**Comment by FahemHakikiKhaya** [comment]
prefix tasknya samain dong CT Dashboard Fixes -

**Comment by hiljaker** [comment]
> prefix tasknya samain dong CT Dashboard Fixes -

siap 🫡

**Review by haikal1223 — APPROVED** [review]

File: src/views/corporate-training/detail/fixed-programs/sections/JadwalKelas/components/IntakeList/IntakeCard.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ interface IntakeCardProps extends ComponentProps<typeof Box> {
   intakeItem: Program
   usingCarousel: boolean
   practiceSessionSchedule: PracticeSession
+  isOneOfBatchesHasPrice: boolean
```
Comment (shadiqfauzi) [inline src/views/corporate-training/detail/fixed-programs/sections/JadwalKelas/components/IntakeList/IntakeCard.tsx:16]: ini condition buat apa ya dan efeknya kemana?

Comment (shadiqfauzi) [reply src/views/corporate-training/detail/fixed-programs/sections/JadwalKelas/components/IntakeList/IntakeCard.tsx:16]: sorry baru baca bawahnya wkwk

**Review by shadiqfauzi — APPROVED** [review]

**Pull request merged**